app/src/main/java/org/simple/clinic/drugstockreminders/DrugStockReminderRequest.kt

```diff
@@ -0,0 +1,11 @@
+package org.simple.clinic.drugstockreminders
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class DrugStockReminderRequest(
```

Comment:
Since we are just passing a date, we can avoid creating a new request type and instead use the `@Query` param in the API function. WDYT?